contracts/OffsetHelper.sol

```diff
@@ -146,6 +146,48 @@ contract OffsetHelper is OffsetHelperStorage {
         autoRetire(tco2s, amounts);
     }
 
```

Comment:
do you think we could rewrite git history to have the tests added within the same commit as the .sol funcs?

Reply:
done